cluster/manifests/kube2iam/rbac.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kube-state-metrics
```

Comment:
This looks wrong. This file mentions both `kube-state-metrics` and `skipper-ingress` but is in the `kube2iam` folder.